packages/node/test/sdk/init.test.ts

```diff
@@ -193,3 +195,65 @@ describe('init()', () => {
     });
   });
 });
+
+describe('validateOpenTelemetrySetup', () => {
+  afterEach(() => {
+    global.__SENTRY__ = {};
+    cleanupOtel();
+    jest.clearAllMocks();
+  });
+
+  it('works with correct setup', () => {
+    const errorSpy = jest.spyOn(logger, 'error').mockImplementation(() => {});
+    const warnSpy = jest.spyOn(logger, 'warn').mockImplementation(() => {});
+
+    jest.spyOn(SentryOpentelemetry, 'openTelemetrySetupCheck').mockImplementation(() => {
+      return ['SentryContextManager', 'SentryPropagator', 'SentrySampler'];
+    });
+
+    validateOpenTelemetrySetup();
+
+    expect(errorSpy).toHaveBeenCalledTimes(0);
+    expect(warnSpy).toHaveBeenCalledTimes(0);
+  });
+
+  it('works with missing setup, without tracing', () => {
+    const errorSpy = jest.spyOn(logger, 'error').mockImplementation(() => {});
+    const warnSpy = jest.spyOn(logger, 'warn').mockImplementation(() => {});
+
+    jest.spyOn(SentryOpentelemetry, 'openTelemetrySetupCheck').mockImplementation(() => {
+      return [];
+    });
+
+    validateOpenTelemetrySetup();
+
+    // Without tracing, this is expected only twice
+    expect(errorSpy).toHaveBeenCalledTimes(2);
+    expect(warnSpy).toHaveBeenCalledTimes(1);
+
+    expect(errorSpy).toBeCalledWith(expect.stringContaining('You have to set up the SentryContextManager.'));
+    expect(errorSpy).toBeCalledWith(expect.stringContaining('You have to set up the SentryPropagator.'));
```

Comment:
l: I think it would be good to also assert on the message we don't expect to show up as well, just relying on the number of times it has been called is error prone.